api/src/main/resources/patientPastAppointments.sql

```diff
@@ -1,10 +1,9 @@
 INSERT INTO global_property (property, property_value, description, uuid)
- VALUES ('bahmni.sqlGet.pastAppointments',
-"SELECT
+VALUES ('bahmni.sqlGet.pastAppointments','SELECT
      app_service.name                                                                                AS `DASHBOARD_APPOINTMENTS_SERVICE_KEY`,
      app_service_type.name                                                                           AS `DASHBOARD_APPOINTMENTS_SERVICE_TYPE_KEY`,
-     DATE_FORMAT(start_date_time, \"%d/%m/%Y\")                                                        AS `DASHBOARD_APPOINTMENTS_DATE_KEY`,
-     CONCAT(DATE_FORMAT(start_date_time, \"%l:%i %p\"), \" - \", DATE_FORMAT(end_date_time, \"%l:%i %p\")) AS `DASHBOARD_APPOINTMENTS_SLOT_KEY`,
```

Comment:
@icrc-toliveira @mks-d what is the mysql version that you are facing error with ?

Reply:
Hi @binduak 
I´m facing this issue with "Mysql  Ver 14.14 Distrib 5.7.31"

Reply:
Tested migration changes with mysql 5.6 Distrib, and the changes are working fine. Approved the PR.